app/dependencies.php

```diff
@@ -304,17 +303,15 @@ function (ContainerInterface $c): Auth\DonationPublicAuthMiddleware {
             return new Serializer($normalizers, $encoders);
         },
 
+        // StripeClientInterface doesn't have enough properties documented to keep
+        // psalm happy.
```

Comment:
I don't think this is just as Psalm issue, interfaces can't really have properties. Yes they could be documented on the interface but there's no system AFAIK to make implementers provide them.

```suggestion
        // Previously we provided StripeClientInterface but we need to use the properties declared by
        // StripeClient. @see https://github.com/stripe/stripe-php/issues/970
```